EcoSoapBankTests/GraphQLControllerTests.swift

```diff
@@ -11,5 +11,228 @@ import XCTest
 
 class GraphQLControllerTests: XCTestCase {
 
+    func testImpactStatsQuerySuccess() {
+        let expectation = XCTestExpectation(description: "Get response from backend API")
+        let graphQLController = GraphQLController()
+        let variables: [String: String] = ["propertyId": "PropertyId1"]
 
+        graphQLController.queryRequest(ImpactStats.self, query: GraphQLQueries.impactStatsByPropertyId, variables: variables) { result in
+
+            guard let result = try? result.get(),
+                let soapRecycled = result.soapRecycled,
+                let linensRecycled = result.linensRecycled,
+                let bottlesRecycled = result.bottlesRecycled,
+                let paperRecycled = result.paperRecycled,
+                let peopleServed = result.peopleServed,
+                let womenEmployed = result.womenEmployed else {
+                    XCTFail("Unable to get valid impact stats from returned data")
+                    return
+            }
+
+            XCTAssert(soapRecycled == 11)
+            XCTAssert(linensRecycled == 11)
+            XCTAssert(bottlesRecycled == 11)
+            XCTAssert(paperRecycled == 11)
+            XCTAssert(peopleServed == 11)
+            XCTAssert(womenEmployed == 11)
```

Comment:
Might be a good idea to change these (and the other similar assertions to use `XCTAssertEqual`); not a huge deal (especially since if they pass it won't make any difference at all) but an easy way to improve consistency.